### ISSUE TITLE
Test set accuracy is very low

### ISSUE THREAD

**Issue by fanyou12** [DoodleJZ/HPSG-Neural-Parser#3]
Loading model from models/joint_bert_dev=95.55_devuas=96.67_devlas=94.86.pt...
Reading dependency parsing data from data/ptb_test_3.3.0.sd
Loading test trees from data/23.auto.clean...
Loaded 2,416 test examples.
Parsing test sentences...
test-fscore (Recall=6.51, Precision=6.70, FScore=6.61) test-elapsed 0h00m34s
best test W. Punct: ucorr: 7738, lcorr: 1426, total: 56684, uas: 13.65%, las: 2.52%, ucm: 1.74%, lcm: 0.75%
best test Wo Punct: ucorr: 7150, lcorr: 1256, total: 49893, uas: 14.33%, las: 2.52%, ucm: 1.82%, lcm: 0.75% 
best test Root: corr: 2416, total: 2416, acc: 100.00%

**Comment by DoodleJZ** [comment]
Thank for showing the issues. Since we update with pytorch_transformers package, it has some problems. Now we use the previous vision of  pretrained_bert and everything is ok. Later, We will find the problems and update the code.
